src/themes/themes.stories.mdx

```diff
@@ -14,8 +14,13 @@ While selector depth is best kept to a minimum, it's permissable to style object
 
 `t-dark` is intended primarily for page headers.
 
+export const parameters = { themes: { disable: true } };
+// Parameters has to be defined separately because of this prettier bug:
+// https://github.com/prettier/prettier/issues/7731
+// https://mdxjs.com/getting-started#defining-variables-with-exports
```

Comment:
@Paul-Hebert I wanted to point out this change in case you didn't see it, since it was not an automated change

Reply:
Cool, thanks. That seems fine.